src/Relude/Extra/Type.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE ExplicitForAll      #-}
```

Comment:
`ExplicitForAll` extension is implied by `ScopedTypeVariables`, so you don't need to write it.